samples/crypto/psa_tls/README.rst

```diff
@@ -234,12 +234,12 @@ Cipher suites not listed may still work but have not been verified.
       * TLS_AES_128_CCM_SHA256
       * TLS_AES_128_CCM_8_SHA256
 
-AES256 is supported on all compatible board targets with the CMSE enabled because of the :ref:`cc3xx_oberon<nrf_security_backends_oberon>` backend.
+AES256 is supported on all compatible board targets with the CMSE enabled because it is supported by both :ref:`nrf_cc3xx and nrf_oberon backends<nrf_security_backends_oberon>`.
```

Comment:
ASE256 is only partly supported by nrf_cc310, which is one of the reasons why we use the combination of the backends nrf_cc3xx and nrf_oberon (cc3xx_oberon). Perhaps we can change the line to something like this:

...  because it enables both backends nrf_cc3xx and nrf_oberon for devices with Arm CryptoCell.